daemon/algod/api/client/restClient.go

```diff
@@ -629,6 +630,12 @@ func (client RestClient) RawDryrun(data []byte) (response []byte, err error) {
 	return
 }
 
+// RawTransactionSimulate gets the raw transaction or raw transaction group, and returns relevant simulation results.
+func (client RestClient) RawTransactionSimulate(data []byte) (response model.SimulateResponse, err error) {
```

Comment:
nit: I'd suggest a name like `SimulateRawTransaction` instead. Functions with the `Raw` prefix in this file seem to return undecoded msgpack response bytes

Reply:
see https://github.com/algorand/go-algorand/pull/5213/commits/478d81629fdefab3996d94b81f09f96e5227e0b3